.gitignore

```diff
@@ -1,3 +1,5 @@
 rootfs/vanilla-pico
 rootfs/*.tar.gz
 Containerfile
+.idea
```

Comment:
I don't think we should add all kinds of IDE specific exceptions here

Reply:
True, I'll add a global .gitignore to my home directory.